### DISCUSSION TITLE
Issue in docs with builtin function examples.

### DISCUSSION THREAD

**Discussion by RmStorm** [open-policy-agent/community#407] [Community]
At the bottom of the extensions doc page [here](https://www.openpolicyagent.org/docs/latest/extensions/#adding-built-in-functions-to-the-opa-runtime).. it seems to me like there is an issue with the registered function. Earlier in that page it says that when making http requests you should set both Memoize and Nondeterministic to true. However in the code I linked Nondeterministic is left out. Is that an error?

I'm not sure if a 'discussion' is the right place to raise this issue but none of the actual issue templates seemed to be right for it.

**Comment by srenatus** [comment]
Sounds reasonable to me! The example predates the Nondeterministic attribute. If you'd like to push a pull request with a fix, it would certainly be appreciated.

**Comment by RmStorm** [comment]
I hadn't realized the website was in the same repo! [PR here](https://github.com/open-policy-agent/opa/pull/5861). I didn't bother completely filling out the template. I hope it's ok.

**Reply by srenatus** [reply]
It's fine! Thank you.
